src/main/resources/templates/db_scripts/transactions.sql

```diff
@@ -1,3 +1,3 @@
 DROP TRIGGER IF EXISTS recalculate_totals on ${myuniversity}_${mymodule}."transaction";
 DROP FUNCTION IF EXISTS ${myuniversity}_${mymodule}.recalculate_totals();
-
+CREATE INDEX IF NOT EXISTS transaction_transactiontype_simple_idx ON ${myuniversity}_${mymodule}.transaction USING btree ((jsonb->>'transactionType'));
```

Comment:
RMB's documentation for `fromModuleVersion` states that:

> subsequent upgrades from versions equal or later than the version indicated for the table will not re-generate the table

so I think we need to create the index outside of the migration updates too, even if it is redundant.

I guess we could also use a script with an empty `fromModuleVersion` to avoid the code redundancy, but then it would get executed even when not needed. If we use a migration and a `customSnippetPath` with the table, I think it is only executed when the table is created or migrated the first time.